extension/screenshooter/src/main/java/org/arquillian/extension/recorder/screenshooter/browser/impl/BrowserScreenshooter.java

```diff
@@ -145,18 +146,18 @@ public Screenshot takeScreenshot(File screenshotToTake, ScreenshotType type) {
         return screenshoot;
     }
 
-    private WebDriver getTakingScreenshotsBrowser() {
-
-        GrapheneContext context = null;
+    private WebDriver getTakingScreenshotsBrowser(File screenshotToTake) {
+        WebDriver result = null;
 
         try {
-            context = GrapheneContext.getContextFor(Default.class);
+            GrapheneContext context = GrapheneContext.getContextFor(Default.class);
+            result = ((GrapheneProxyInstance) context.getWebDriver(TakesScreenshot.class)).unwrap();
```

Comment:
But the Graphene context "keeps" the WebDriver (proxy), and when the method GrapheneContext.getWebDriver(...) is called it tries to retrieve it. And this is the moment when the ISE ("There is no context available for qualifier org.jboss.arquillian.drone.api.annotation.Default. Available contexts are []") should be thrown (for the incontainer tests). 
I don't know why, but I forgot the annotation `@RunAsClient` in my test case. Here is the fix: d5479c6d3dec76fffea1b5810251c41a40409005
This test passes without failing. If you have different observation, please share the reproducer. Thanks